chain-signatures/node/src/web/mod.rs

```diff
@@ -28,8 +28,8 @@ pub async fn run(
     sender: Sender<Ciphered>,
     protocol_state: Arc<RwLock<NodeState>>,
     indexer: Indexer,
-) -> anyhow::Result<()> {
-    tracing::info!("running a node");
+) {
+    tracing::info!("starting web server");
```

Comment:
Can we keep "node" here? We may have other web servers that are not nodes. And we already have Redis.